descriptions.yml

```
@@ -2,4 +2,303 @@
   Criterias query entities.
 
 ".paths./salutation.post": >
-  Fetches salutations with a criteria obj.
\ No newline at end of file
+  Fetches salutations with a criteria obj.
+  
+".components.schemas.Product.properties.versionId":>
```

```suggestion
".components.schemas.Product.properties.versionId": >
```
Please add the spaces in the rest of the document